app.js

```diff
@@ -11,10 +11,17 @@ app.post('/agendas/:agenda_id/agendaitems/documents/files/archive', async (req,
   let files;
   if (mandateeIdsString) {
     const mandateeIds = mandateeIdsString.split(',');
+    if (req.query.decisions){
```

Comment:
I suspect `req.query.decisions` will contain the string `true` or `false` (given that `URLSearchParams` is correctly used in the frontend). You will want to explicitly check `if (req.query.decisions === 'true')` here.

Reply:
fixed